README.md

```diff
@@ -59,36 +59,45 @@ We are breaking down the various parts that we need to migrate to Swift in the s
-|                     | `setStickyFeatures`                                 |        |
-|                     | `getRevision`                                       |        |
-|                     | `getFeature`                                        |        |
-|                     | `getBucketBy`                                       |        |
-|                     | `getBucketValue`                                    |        |
-|                     | `isReady`                                           |        |
+|                     | `onReady`                                           | ✅     |
+|                     | `onRefresh`                                         | ✅     |
+|                     | `onUpdate`                                          | ✅     |
+|                     | `refreshInternal`                                   | ✅     |
+|                     | `stickyFeatures`                                    | ✅     |
+|                     |                                                     |        |
+| Instance methods    | `constructor` missing fetch datafile content        | ⚠️      |
+|                     | `setDatafile` removed to workaround init issues     | ✅     |
+|                     | `setStickyFeatures`                                 | ✅     |
+|                     | `getRevision`                                       | ✅     |
+|                     | `getFeature`                                        | ✅     |
+|                     | `getBucketKey`                                      | ✅     |
+|                     | `getBucketValue`                                    | ✅     |
+|                     | `isReady`                                           | ✅     |
 |                     | `refresh`                                           |        |
 |                     | `startRefreshing`                                   |        |
 |                     | `stopRefreshing`                                    |        |
-|                     | `evaluateFlag`                                      |        |
-|                     | `isEnabled`                                         |        |
 |                     | `evaluateVariation`                                 |        |
-|                     | `getVariation`                                      |        |
+|                     | `getVariation`                                      | ✅     |
+|                     | `getVariationBoolean`                               |        |
+|                     | `getVariationString`                                |        |
+|                     | `getVariationInteger`                               |        |
+|                     | `getVariationDouble`                                |        |
 |                     | `activate`                                          |        |
+|                     | `activateBoolean`                                   |        |
```

Comment:
because `getVariation()` only returns a string going forward, type specific methods for activate() not no longer needed.

Reference: https://github.com/fahad19/featurevisor/pull/105